src/Raven.Server/Documents/Expiration/ExpiredDocumentsCleaner.cs

```diff
@@ -27,6 +27,10 @@ public class ExpiredDocumentsCleaner : BackgroundWorkBase
             ? 4096
             : 1024;
 
+        internal static int MaxItemsToProcess = PlatformDetails.Is32Bits == false
```

Comment:
1. Please rename `MaxItemsToProcess` to `DefaultMaxItemsToProcessInSingleRun`.
2. Let's avoid changing the default behavior and put `int.MaxValue` as the default for both x86 and x64. I understand the motivation but I'm a bit worried that the feature will start behaving differently on 32bits

Reply:
done